aiohttp/client.py

```diff
@@ -212,7 +212,8 @@ class ClientTimeout:
 IDEMPOTENT_METHODS = frozenset({"GET", "HEAD", "OPTIONS", "TRACE", "PUT", "DELETE"})
 HTTP_SCHEMA_SET = frozenset({"http", "https", ""})
 WS_SCHEMA_SET = frozenset({"ws", "wss"})
-ALLOWED_PROTOCOL_SCHEMA_SET = HTTP_SCHEMA_SET | WS_SCHEMA_SET
+PROTOCOL_SCHEMA_SET = frozenset({"unix"})
```

Comment:
Ideally this would only be when using the `UnixConnector` but thats a breaking change, and the goal for the moment is to fix the regression.

https://github.com/aio-libs/aiohttp/issues/8633 will track moving this to be per connector

Reply:
This will still leave NamedPipeConnector broken, right? I think we should delay a couple of days if needed to get this defined properly in the connectors.